docs/reference/api.html.md

```diff
@@ -2152,6 +2241,22 @@ Unknown: For some reason the state of the Argo CD Dex component could not be obt
 </tr>
 <tr>
 <td>
+<code>notifications</code></br>
```

Comment:
<img width="720" alt="image" src="https://user-images.githubusercontent.com/46771830/170509487-b3b4ea20-5e04-4c45-b3c8-b8a38ed8c6c7.png">

should we add more space for the column to fit notifications in 1 line 🤔 ?

Reply:
done, thanks @reginapizza !